docs/pages/per_address_priority.md

```diff
@@ -9,7 +9,7 @@ To better handle multiple source control scenarios, the 0xDD sACN start code was
 The following acronyms and terms are used:
 - *sACN*,  *Streaming ACN*, *Streaming DMX*: Reference “BSR E1.31 DMX512-A Streaming Protocol”
 - *Receiver*: A device that receives sACN
-- *Source*: A device that send sACN. Sources are uniquely identified by their CID number.
+- *Source*: A device that sends sACN. Sources are uniquely identified by their CID number.
```

Comment:
Just "CID", not "CID number", I think.

Reply:
https://github.com/ETCLabs/sACN/pull/38/commits/4c389d544ed47fe801f15b99dd1b463d418fa773